### PR TITLE
Best Practices

### PR DESCRIPTION
**Pull request by anshul-07-garg**

### CODE REVIEW

File: restapi/views.py [hunk 1]
```diff
@@ -186,22 +214,23 @@ def logProcessor(request):
     log_files = data['logFiles']
     if num_threads <= 0 or num_threads > 30:
         return Response({"status": "failure", "reason": "Parallel Processing Count out of expected bounds"},
-                        status=status.HTTP_400_BAD_REQUEST)
+                        status=HttpStatusCode.BAD_REQUEST.value)
     if len(log_files) == 0:
         return Response({"status": "failure", "reason": "No log files provided in request"},
-                        status=status.HTTP_400_BAD_REQUEST)
+                        status=HttpStatusCode.BAD_REQUEST.value)
     logs = multiThreadedReader(urls=data['logFiles'], num_threads=data['parallelFileProcessingCount'])
```
[the remark this answers is not in the document]
Comment (anshul-07-garg) [reply restapi/views.py]: Done.

File: restapi/urls.py [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
 
 
 router = DefaultRouter()
-router.register('users', user_view_set)
-router.register('categories', category_view_set)
-router.register('groups', group_view_set)
-router.register('expenses', expenses_view_set)
+router.register('users', USER_VIEW_SET)
+router.register('categories', CATEGORY_VIEW_SET)
+router.register('groups', GROUP_VIEW_SET)
```
[the remark this answers is not in the document]
Comment (anshul-07-garg) [reply restapi/urls.py]: Changed to TitleCase

File: restapi/custom_exception.py [hunk 3]
```diff
@@ -1,7 +1,20 @@
 from rest_framework.exceptions import APIException
+from enum import Enum, unique
+
+
+@unique
+class HttpStatusCode(Enum):
```
[the remark this answers is not in the document]
Comment (anshul-07-garg) [reply restapi/custom_exception.py]: Used the inbuilt enum.

**Comment by kushal-ti** [comment]
looks good

**Pull request closed without merging**